CHANGELOG.md

```diff
@@ -82,6 +82,11 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
   rounding issues. This is primarily to support things like `avg(int_col)`,
   which we define as returning `Numeric`
 
+* `#[check_types]` has been added to help debug missing `Queryable` impls. See
```

Comment:
I don't think this is right

Reply:
What's wrong with it?

Reply:
They're not missing, just wrong impls

Reply:
I worded it this way because the message the user gets is literally "`Queryable<...>` is not implemented"

Reply:
Makes sense I guess